pkg/features/kube_features.go

```diff
@@ -631,6 +631,7 @@ const (
 	// owner: @ehashman
```

Comment:
feel free to put your name here for the history. Thank you for moving it forward